app/views/foods/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @search_result_count >= 10000 %>
```

Comment:
So there was a `totalHits` key that comes back from this endpoint, and that's what you should have used as the value here.